scripts/cxg_admin.py

```diff
@@ -116,7 +116,10 @@ def transfer_collections(ctx, curr_owner, new_owner):
                 .filter(DbCollection.owner == curr_owner)
```

Comment:
I think this needs to be unpacked             
```updated = (
                *session.query(DbCollection)
                .filter(DbCollection.owner == curr_owner)
                .update({DbCollection.owner: new_owner})
            )
```

It can be shorter:
```
update = (*collections.update({DbCollection.owner: new_owner})

Reply:
its shorter but I think it loses some clarity about what is actually happening